functions/Copy-SqlLogin.ps1

```diff
@@ -356,6 +366,15 @@ Limitations: Does not support Application Roles yet
 				{
 					Update-SqlPermissions -sourceserver $sourceserver -sourcelogin $sourcelogin -destserver $destserver -destlogin $destlogin
 				}
+				if ($rename.Keys -contains $username) { 
```

Comment:
Very nice! You really simplified this task. Can you do the same for Databases? ;) mmm that one would be harder, though because of the files inside, boo.

Please rename Update-SqlLoginName to Update-DbaLoginName -- both within the file and within the filename. 

Can you please research the variable name that people usually use for a rename hash? I'm guessing it would be LoginRenameHash? Something more specific than Hashtable.

Otherwise, dope AF! Thank you 🥇